src/types/Storage/methods/updateAddress.js

```diff
@@ -60,15 +60,13 @@ const updateAddress = function (addressObj, walletId) {
       utxo.outputIndex = parseInt(outputIndex, 10);
 
       try {
-        const { blockHeight } = this.getTransaction(txid);
-        if (currentBlockHeight - blockHeight >= 6) newObject.balanceSat += utxo.satoshis;
-        else newObject.unconfirmedBalanceSat += utxo.satoshis;
+        this.getTransaction(txid);
+        // TODO : We removed here the confirmations verification
+        // We should ensure we had a locked block before being able to really spend those.
+        newObject.balanceSat += utxo.satoshis;
       } catch (e) {
-        logger.error('Error', e);
-        if (e instanceof TransactionNotInStore) {
-          // TODO : We consider unconfirmed a transaction that we don't know of, should we ?
-          newObject.unconfirmedBalanceSat += utxo.satoshis;
-        } else throw e;
+        logger.error(`Unable to find locally ${txid}`);
+        if (!(e instanceof TransactionNotInStore)) throw new Error(e);
```

Comment:
Is `e` not an instance of `Error` ?

Reply:
TransactionNotInStore is Error extended

Reply:
Then we don't need to wrap it in another `Error` then?

Reply:
Fixed. Also verified it's not elsewhere